version-migration/v0.6-to-v0.7-migration/examples/src/test/kotlin/examples/BreakingChanges.kt

```diff
@@ -0,0 +1,9 @@
+package examples
+
+import kotlin.test.Test
+
+class BreakingChanges {
+    @Test
+    fun `api change - removal of the old LIKE operator classes`() {
+    }
```

Comment:
What's the strategy for removal of classes? Do you want to import them to show which are removed?

Reply:
For this particular case, I looked further into the removed classes, but they were all `internal` in v0.6.0, so this isn't a breaking change that needs documented. I'll update the migration guides and entry in the `CHANGELOG.md`.

Usually for removed public APIs, they should have been deprecated previously and should have an alternative non-deprecated API that should be used. I think it would be helpful to show examples with the previous, deprecated API and the migrated versions using the new API.

Reply:
ACK. Thanks! I see that 0.7 also removed a couple other things. If they're public, can you show those in this file?

Reply:
Yeah, updated to include an example for the changed `ErrorCode`. The other breaking API changes was an old `ErrorCode` and class that were only used in testing and not exposed via any other APIs.